BBHX_Phenom.py

```
@@ -231,6 +245,7 @@ def _bbhx_fd(
     else:
         err_msg = f"Don't recognise reference frame {ref_frame}. "
         err_msg = f"Known frames are 'LISA' and 'SSB'."
+        ValueError(err_msg, RuntimeWarning)
```

This isn't quite right, you need to raise the error, e.g. `raise ValueError(<message>)`